src/actions/actions.ts

```diff
@@ -610,6 +610,18 @@ class CommandFormatCode extends BaseCommand {
 
   public async exec(position: Position, vimState: VimState): Promise<VimState> {
     await vscode.commands.executeCommand("editor.action.format");
+
+    let selection = vscode.window.activeTextEditor.selection;
```

Comment:
Is there a reason you aren't using vimState.cursorPosition and cursorStartPosition?